locale/es.yml

```diff
@@ -0,0 +1,15 @@
+clarkwinkelmann-emojionearea:
+    forum:
+        picker_button: Selector de Emojis
+        search_placeholder: Búsqueda
+
+    admin:
+        settings:
+            title: Configuración de EmojioneArea
+            enableSearch: Activar búsqueda
+            enableRecent: Activar recientes
+            enableTones: Activar matices
+            filtersPosition: Mover categorías abajo
+            searchPosition: Mover búsqueda y matices abajo
+            hideFlarumButton: Ocultar el botón nativo Emoji de Flarum
+hideFlarumButtonHelp: La extensión Emoji Picker debe estar activada junto con la extensión nativa de Emoji para que esta funcione
```

Comment:
Thanks for the contribution. There's an indentation issue here, but I can fix it when I'll merge tomorrow :wink:

Reply:
> Thanks for the contribution. There's an indentation issue here, but I can fix it when I'll merge tomorrow 😉

Oh, my bad! I'm new around here.
You are more than welcome, I'm glad I could help. 😄